Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml

```
@@ -130,6 +129,9 @@
   - type: Sprite
     state: chocolatebar-open
   - type: Item
+  - type: Tag
+    tags:
+      - ReptilianFood
```

```suggestion
    tags:
      - FoodSnack
      - ReptilianFood
```
FoodSnackChocolateBar inherits the FoodSnack tag from FoodSnackBase. Since tags get overwritten and not merged if you redefine them, you have to add that tag here as well, so that it does not get removed.